content/docs/1.0.x/reference/api/index.md

```diff
@@ -59,7 +62,34 @@ The Keptn API is documented in terms of a [Swagger API documentation](https://sw
         link="./assets/swagger-example.png"
         caption="Keptn Swagger API documentation - Example"
         width="700px">}}
-
+        
+## Upload a file to a Keptn project
+You can upload an API file to a project in any of the following ways :
+* Use the [keptn add-resource](../../cli/commands/keptn_add-resource) command.
+Note that only some APIs are supported by this command.
+* Upload the file directly to your Git repo.
+* Use the `resource-service` API for uploading
+using the Swagger API as shown above.
+This gives you sections for uploading resources
+on the project, stage or service level.
+
+  To mimic what the `keptn add-resource` command does,
+  use the `POST API` call.
+  
+  ```console
+  curl -X 'POST' \
+  'http://yourkeptn/api/resource-service/v1/project/YOURPROJECT/stage/YOURSTAGE/service/YOURSERVICE/resource' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "resources": [
+    {
+      "resourceContent": "PATHTO/yourfile.abc",
+      "resourceURI": "BASE64ENCODEDFILE"
+    }
+  ]
+  }
+  ```
 ## Architecture Details of Keptn API
 
 Keptn 0.6.2 introduced an NGINX as new K8s deployment and service called `api-gateway-ngnix` shown in the figure below. This NGINX allows to route the traffic and ensures that all requests are authenticated using the `/auth` endpoint of the `api-service`.
```

Comment:
```suggestion
Keptn uses
[NGINX](https://www.nginx.com/)
to route the traffic and ensure that all requests are authenticated
using the `/auth` endpoint of the `api-service`.
This is implemented using the `api-gateway-ngnix` service
as shown in the figure below.
```
The history is no longer relevant but the implementation is of interest.  I also broke this up into shorter lines in case someone turns on markdownlint for this repo.

Reply:
The bullet list that followed this text also needed updating -- it talks about the configuration-service, which was replaced by the resource-service in 0.16.x.  I couldn't get to that text to make "suggestions" so just did an edit and committed it to your branch.  While I was at it, I linked to the main Architecture document for some stuff and I hope I got the right number of ../ strings in -- I can't check xrefs when using the github editor.  I guess the xref checker will flag them if I got them wrong.

Reply:
okay, Except for this, I guess everything is okay! What's your point of view? @StackScribe